utils.js

```diff
@@ -53,3 +55,6 @@ module.exports = {
     return out;
   },
 };
+
+module.exports.isOpenApiJson = isOpenApiJson;
```

Comment:
nit:

wdyt of making these exports more obvious that they are test helper methods?

prefixing/suffixing the function names with `test`?

or exporting a `testables` object (a pattern I've used in prior projects):

```js
module.exports.testables = { isOpenApiJson, isOpenApiYaml };
```